app-authz-rest-springboot/pom.xml

```diff
@@ -45,7 +47,20 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.2</version>
+            <version>${httpclient.version}</version>
```

Comment:
Why are all these dependency changes needed? Does this indicate some breaking changes in our POMs?